### ISSUE TITLE
CLB: unrepairable sectors due to missing targets

### ISSUE THREAD

**Issue by yroskov** [CatalogueOfLife/backend#1294]
Working with 3i Auchenorrhyncha... https://github.com/CatalogueOfLife/testing/issues/250#issuecomment-1952663642 

Step 1: I am checking sectors in 3i Auchenorrhyncha: all sectors are not broken in the report, however, **target is missing** in all these sectors:
https://www.checklistbank.org/catalogue/3/sector?limit=100&offset=0&subjectDatasetKey=2317
![image](https://github.com/CatalogueOfLife/testing/assets/28574252/b140fd2a-3ab4-4085-85f9-c8fd33300046)
![image](https://github.com/CatalogueOfLife/testing/assets/28574252/1ce212d5-8951-48d9-b30d-9f7d240c39d8)

![image](https://github.com/CatalogueOfLife/testing/assets/28574252/97156528-f8f2-4a54-95a4-363adf3048b3)
![image](https://github.com/CatalogueOfLife/testing/assets/28574252/e98f47d0-143a-47a9-9ffb-14d9feb3c45e)

**I cannot repair sectors because targets "suborder Auchenorrhyncha" & "infraorder Cicadomorpha" absent in target search.** 

Step 2: I am checking presence of target taxa in Assembly Tree: **both taxa are present there** 
https://www.checklistbank.org/catalogue/3/assembly?assemblyTaxonKey=43Pi9-r6k2QQ5t0klrv2u1&datasetKey=288268&sourceTaxonKey=320
![image](https://github.com/CatalogueOfLife/testing/assets/28574252/ea223caf-3738-4dff-919d-b3cf487f2429)

@mdoering, could you please help? Perhaps, this caused by indexing problem...

**Comment by mdoering** [comment]
reindexed: https://api.checklistbank.org/dataset/2317/nameusage/search?q=Cicadomorpha&rank=infraorder

**Comment by mdoering** [comment]
Oh, target is missing. That means reindexing of the entire project which takes 1-2h and makes it rather unusable during that time. Shall I?

**Comment by mdoering** [comment]
btw, slack would be a quicker communication for these kind of problems

**Comment by yroskov** [comment]
>Shall I?

Yes, pls. I'll keep my fingers out of CLB until you give me green light

**Comment by yroskov** [comment]
OK, I am in Slack now. Waiting for your further comments
